lib/generators/shopify_app/user_model/user_model_generator.rb

```diff
@@ -16,6 +16,27 @@ def create_user_migration
         migration_template('db/migrate/create_users.erb', 'db/migrate/create_users.rb')
       end
 
+      def create_scopes_storage_in_user_model
+        scopes_column_prompt = <<~PROMPT
+          It is highly recommended that apps record the access scopes granted by \
+          merchants during app installation. See app/models/user.rb to modify how \
+          access scopes are stored and retrieved.
+
+          [WARNING] There may be unintended behaviour during the OAuth flow \
+          if access_scopes are not stored.
```

Comment:
This feels a little vague without the context of what exactly might fail, can we be a bit more specific without making the message confusing?

Reply:
Updated the messaging around this.